azure_arc_k8s_jumpstart/aksengine/arc_k8s_aksengine.azcli

```diff
@@ -0,0 +1,432 @@
+########################################################
+# Script to deploy different features of Arc-enabled
+#    k8s clusters, for demo or training purposes.
+#
+# Jose Moreno
+# March 2021
+########################################################
+
+
+#############################
+# Deploy aks-engine cluster #
+#############################
+
+# Variables
+k8s_rg=aksengine
```

Comment:
Generalize the variable to something like location='<Azure Region>', that way there is a description and the user can decide. I suggested some changes in the _index.md to describe what the variables are

Reply:
What do you mean? the variable is called "location"